pythoncalculator/divide.py

```diff
@@ -0,0 +1,2 @@
+def divide(x, z):
+    return x / y
```

Comment:
There is an error in this code, please use the suggestion to change it.
```suggestion
    return x / z
```